src/components/Dropdown/Option.ts

```diff
@@ -1,4 +1,6 @@
 export default interface Option {
   label: string
-  disabled: boolean | undefined
+  href?: string
+  selected?: boolean
+  disabled?: boolean
```

Comment:
I don't like the idea of setting `href` & `selected`. May we use `label` and `value` instead

Reply:
Let's name it `forceSelected`